std_msgs/String topic_name
std_msgs/String target_frame
---
sensor_msgs/PointCloud2 cloud_out